src/cli.cr

```diff
@@ -30,6 +30,7 @@ module Shards
       opts.on("--no-color", "") { self.colors = false }
       opts.on("--version", "") { puts self.version_string; exit }
       opts.on("--production", "") { self.production = true }
+      opts.on("--dev", "") { self.dev = true }
```

Comment:
No abbreviations please, this should be `--development`.

I'm uncomfortable having both `--production` (don't install development dependencies) and `--development` (add development dependency to `shard.yml` as global options, but for the time being individual commands didn't need special flags, yet. Maybe an evolution should allow this (later).